docs/docs/features/underglow.md

```diff
@@ -34,41 +34,47 @@ CONFIG_ZMK_RGB_UNDERGLOW=y
+If some of the LEDs on your keyboard don't light up, that could be a hardware problem. But another possibility is that the number of LEDs in the default configuration doesn't match the number that are installed. For example, the `corne` shield specifies only 10 LEDs per side while you might have as many as 27. On a split keyboard, a good sign of this problem is if the lit LEDs on each half are symmetrical.
 
-The number of underglow LEDs is controlled by the `chain-length` property in the `led_strip` node. You can [change the value of this property](../config/index.md#changing-devicetree-properties) in the `<keyboard>.keymap` file by adding a stanza like this one outside of any other node (i.e. above or below the `/` node):
+The `chain-length` property of the `led_strip` node controls the number of underglow LEDs. If it is incorrect for your build, [you can change this property](../config/index.md#changing-devicetree-properties) in your `<keyboard>.keymap` file by adding a stanza like this one outside of any other node (i.e. above or below the `/` node):
 
 ```dts
 &led_strip {
     chain-length = <21>;
 };
 ```
 
-where the value is the total count of LEDs (per half, for split keyboards).
+For split keyboards, set `chain-length` to the number of LEDs installed on each half.
 
 ## Configuring RGB Underglow
 
 See [RGB underglow configuration](/docs/config/underglow).
 
 ## Adding RGB Underglow to a Board
 
-RGB underglow is always added to a board, not a shield. This is a consequence of needing to configure SPI to control the LEDs.
-If you have a shield with RGB underglow, you must add a `boards/` directory within your shield folder to define the RGB underglow individually for each board that supports the shield.
-Inside the `boards/` folder, you define a `<board>.overlay` for each different board.
-For example, the Kyria shield has a `boards/nice_nano.overlay` file that defines the RGB underglow for the `nice_nano` board specifically.
+Support for RGB underglow is always added to a board, not a shield. This is because the LED strip drivers rely on hardware-specific interfaces (e.g. SPI, I2S) and configurations, which shields do not control.
+
+Shields written for boards which support RGB underglow should add a `boards/` folder underneath the shield folder. Inside this `boards/` folder, create a `<board>.overlay` for any of the boards the shield can be used with. Place all hardware-specific configurations in these `.overlay` files.
+
+For example: the `kyria` shield has a [`boards/nice_nano.overlay`](https://github.com/zmkfirmware/zmk/blob/main/app/boards/shields/kyria/boards/nice_nano.overlay) and a [`boards/nrfmicro_13.overlay`](https://github.com/zmkfirmware/zmk/blob/main/app/boards/shields/kyria/boards/nrfmicro_13.overlay), which configure a WS2812 LED strip for the `nice_nano` and `nrfmicro_13` boards respectively.
```

Comment:
```suggestion
For example: the `kyria` shield has a [`boards/nice_nano_v2.overlay`](https://github.com/zmkfirmware/zmk/blob/main/app/boards/shields/kyria/boards/nice_nano_v2.overlay) and a [`boards/nrfmicro_13.overlay`](https://github.com/zmkfirmware/zmk/blob/main/app/boards/shields/kyria/boards/nrfmicro_13.overlay), which configure a WS2812 LED strip for the `nice_nano_v2` and `nrfmicro_13` boards respectively.
```
Nit, I usually prefer v2 (as the more frequent target) to reduce potential mix-ups if people copy paste things like this.